samples/lightdb_led/README.rst

```diff
@@ -171,16 +187,6 @@ This is the output from the serial console:
    [00:00:00.040,000] <inf> golioth_lightdb: LED 3 -> 1
    [00:00:00.040,000] <inf> golioth_lightdb: LED 0 -> 1
 
-Monitor counter value
-=====================
-
-Device increments counter every 5s and updates ``/counter`` resource in Light DB
-with its value. Current value can be fetched using following command:
-
-.. code-block:: console
-
-   goliothctl lightdb get <device-id> /counter
-
```

Comment:
Good catch. Deserves separate commit though. It is also a good idea to reference commit ID of the change that removed counter usage in code, so that it is easy to track down when documentation went out of sync. This is quite useful for maintenance of stable releases (which we don't do yet, but there are plans to start doing them).